src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12649,7 +12653,8 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
 
     // set supplemental heating coil calculation if the condition requires
     if (this->SuppHeatingCoilPresent) {
-        if ((QZnReq > DataHVACGlobals::SmallLoad && QZnReq > FullOutput) ||
+        if (((QZnReq > DataHVACGlobals::SmallLoad && QZnReq > FullOutput) ||
+             (((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad) && QZnReq <= 0.0)) ||
```

Comment:
This logic determines the conditions when supplemental heating coil needs to run.

Reply:
Same comment here. Is this change necessary?